styles/scss/pivotTable.scss

```diff
@@ -402,7 +402,8 @@ $header-cell-resize-width: 20px;
             border-top-color: white;
 
             &.gd-hidden-sticky-column,
-            &.gd-measure-column {
+            &.gd-measure-column,
+            &.gd-column-attribute-column{
```

Comment:
Only this line is solving the root cause of BB-1463, the rest is related to the horizontal scrolling